java.lang.NullPointerException at org.osmdroid.config.DefaultConfigurationProvider.load

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3270)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3409)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:83)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2016)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7356)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:491)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:940)
Caused by: java.lang.NullPointerException: 
  at org.osmdroid.config.DefaultConfigurationProvider.load (DefaultConfigurationProvider.java:337)
  at org.tosl.coronawarncompanion.MainActivity.onCreate (MainActivity.java:237)
  at android.app.Activity.performCreate (Activity.java:7824)
  at android.app.Activity.performCreate (Activity.java:7813)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1306)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3245)
```

(report from Google Play; 4 users reported a total of 25 occurrences, 1 user reported 22 occurrences)

@uhengart you might want to have a look...

--- a/corona-warn-companion/build.gradle
+++ b/corona-warn-companion/build.gradle
@@ -128,7 +128,7 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-scalars:2.9.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
     implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
-    implementation 'org.osmdroid:osmdroid-android:6.1.8'
+    implementation 'org.osmdroid:osmdroid-android:6.1.6'
 
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.10'
 

